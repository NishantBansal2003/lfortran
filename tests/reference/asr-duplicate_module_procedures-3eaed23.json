{
    "basename": "asr-duplicate_module_procedures-3eaed23",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/duplicate_module_procedures.f90",
    "infile_hash": "e9cde8ba3a436d435bbef78838652ac5e8b74afc461c5136c192897a",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-duplicate_module_procedures-3eaed23.stderr",
    "stderr_hash": "411c842caeeb2d38d64876b95e787e9929f0ba90773ee6abf14fcb8a",
    "returncode": 2
}