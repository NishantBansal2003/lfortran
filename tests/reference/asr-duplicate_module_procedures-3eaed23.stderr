semantic error: Entity test_01 is already present in the interface
  --> tests/errors/duplicate_module_procedures.f90:8:5 - 12:26
   |
 8 |        SUBROUTINE test_01 (x)
   |        ^^^^^^^^^^^^^^^^^^^^^^...
...
   |
12 |        END SUBROUTINE test_01
   | ...^^^^^^^^^^^^^^^^^^^^^^^^^^ 
